CapstoneProject/Assets/Scripts/Core/BehaviorTree/AI.cs

```diff
@@ -308,6 +312,7 @@ void AnimStateUpdate()
 
                 if (enemy.GetComponent<EnemyHealthComponent>().isDead == true)
                 {
+                    _anim.SetInteger("state", -1);
                     _anim.SetTrigger("death");
                 }
```

Comment:
This exact if statement is in each of the cases for enemy type, should we just do it outside of the switch once?